tests/test_rest_api.py

```diff
@@ -12,15 +12,15 @@ class TestRestApiWithSwarm:
     @pytest.mark.asyncio
     @pytest.mark.parametrize("peer", random.sample(nodes_with_auth(), 1))
     async def test_hoprd_rest_api_should_reject_connection_without_any_auth(self, swarm7: dict[str, Node], peer: str):
-        url = f"http://{swarm7[peer].host_addr}:{swarm7[peer].api_port}/api/v3/node/version"
+        url = f"http://{swarm7[peer].host_addr}:{swarm7[peer].api_port}/api/v4/node/version"
```

Comment:
How about we do the same thing here 
<HOST>:<PORT>/api/v3/node/version => <HOST>:<API>/version
as it was done with with 
<HOST>:<API>/readyz

Reply:
That is very attacked friendly. 🍻